integrations/access/accessrequest/app.go

```diff
@@ -352,16 +352,23 @@ func (a *App) getMessageRecipients(ctx context.Context, req types.AccessRequest)
 		recipientSet.Add(common.Recipient{})
 		return recipientSet.ToSlice()
 	case types.PluginTypeOpsgenie:
-		if recipients, ok := req.GetSystemAnnotations()[types.TeleportNamespace+types.ReqAnnotationSchedulesLabel]; ok {
-			for _, recipient := range recipients {
-				rec, err := a.bot.FetchRecipient(ctx, recipient)
-				if err != nil {
-					log.Warning(err)
-				}
-				recipientSet.Add(*rec)
+		// For backwards compatibility we need to use the schedules label to grab recipients
+		// if the notify services isn't set.
+		recipients, ok := req.GetSystemAnnotations()[types.TeleportNamespace+types.ReqAnnotationNotifyServicesLabel]
+		if !ok {
+			recipients, ok = req.GetSystemAnnotations()[types.TeleportNamespace+types.ReqAnnotationSchedulesLabel]
+			if !ok {
+				return recipientSet.ToSlice()
+			}
+		}
```

Comment:
Didn't we agree that in master (v16) each annotation will be used for its intended purpose, and in backports we'll use "schedules" for both if it's the only one present? From our discussion:

```
- In v15/v14 to keep backwards compatibility the behavior should be:
  - When only [teleport.dev/schedules](http://teleport.dev/schedules) annotation is present, it is used for both notifications and auto-approval to keep the behavior there is now. This behavior should not be documented anywhere, this is just for compatibility because the mess we made.
  - When both notify-services and schedules are present, each is used for their own intended purpose.
- In master (v16), we keep notify-services and schedules for their own purposes without this side-effect of sending notifications. I guess this means you'll need to revert the revert of your PR again because that's what basically your master PR already did.
  - And add this to breaking changes in the changelog to tell users they must have "notify-services" defined to receive notifications in Opsgenie.
- Go through all Opsgenie documentation and RFDs and make sure all annotations everywhere are accurate and properly documented, including screenshots.
```